music21/duration.py

```diff
@@ -2929,6 +2938,7 @@ def type(self) -> str:
         0.25
         '''
         if self.linked is False:
+            assert self._unlinkedType is not None, 'linked setter failed to set _unlinkedType'
```

Comment:
I'm not sure what this is for; I'm again not happy with these bare asserts -- it doesn't tell the user what they should do if they see this message.  I'd rather have a mypy ignore than to have extra asserts.